setup-gems-cache/action.yml

```diff
@@ -0,0 +1,112 @@
+name: Setup Gems Cache
+description: Setup Gems Caching for workflow
+inputs:
+  ci_test_mode: 
+    description: Determine the composite action behavior on normal and ci test mode
+    required: false
+    default: '0'
+  unlink: 
+    description: Enable unlink specific behavior on the composite action
```

Comment:
Could you make the be a bit more specific about what these options do?

Reply:
We might need an explanation from casks developer about this.